libs/core/langchain_core/messages/utils.py

```diff
@@ -418,6 +419,11 @@ def filter_messages(
             SystemMessage, HumanMessage, AIMessage, ...). Default is None.
         include_ids: Message IDs to include. Default is None.
         exclude_ids: Message IDs to exclude. Default is None.
+        exclude_tool_calls_id: Tool call IDs to exclude. Default is None.
+            If `True`, all messages with tool calls will be excluded.
+            It filters pairs of AIMessage with the invocation
```

Comment:
this needs to be updated to clarify that the tool calls are filtered/updated in the AI message